nautobot/dcim/models/power.py

```diff
@@ -338,3 +339,15 @@ def get_occupied_positions(self) -> set[int]:
 
     def get_type_class(self):
         return PowerFeedTypeChoices.CSS_CLASSES.get(self.type)
+
+    @property
+    def get_utilization(self):
```

Comment:
Can this be just `utilization` or else not be a `@property`? `get_*` with a property name feels wrong to me.